### PR TITLE
Nolabeloverride

### PR DESCRIPTION
**Pull request by gustaphe**
Without this PR,
```julia
y = (1:10)u"m"
plot(y; yguide="y")
plot!(-y)
```
would have the `yguide` `m`, because the `plot!` call would only check if `yguide` was given as an argument, so you would need to make sure to set the label in your last plot call. This is in contrast to how unitless labels and most other keywords work.

With this PR,
```julia
y = (1:10)u"m"
plot(y; yguide="y")
plot!(-y)
```
has `yguide` `y (m)`.

### CODE REVIEW

**Review by briochemc — COMMENTED** [review]
Thanks! LGTM 😃 Could you also add a screenshot of before/after of your test example? And bump the last digit in the version number?

**Comment by gustaphe** [comment]
before
after

**Review by briochemc — APPROVED** [review]

**Comment by briochemc** [comment]
LGTM! Let's wait for @jw3126's green light and then we can merge this and tag a new release! Thanks again!

**Comment by jw3126** [comment]
LGTM! Thanks! Your honor to merge @gustaphe

**Pull request merged**
